docs/maps/search.asciidoc

```diff
@@ -2,35 +2,69 @@
 [[maps-search]]
 == Search geographic data
 
-Search across the layers in your map to focus in on just the data you want.
+Search across the layers in your map to focus on just the data you want. Combine free text search with field-based search using the <<kuery-query,{kib} Query Language>>. Set the time filter to restrict layers by time.
 
-Layers that request data from {es} are narrowed when you submit a <<kibana-concepts-searching-your-data, search request>>.
+This image shows an example of global search and global time filter narrowing results.
+
+[role="screenshot"]
+image::maps/images/global_search_bar.png[]
```

Comment:
add comment between [] for accessibility:

[ global search and filter narrowing results]